projects/plugins/jetpack/modules/comment-likes.php

```diff
@@ -105,16 +118,22 @@ function enqueue_admin_styles_scripts() {
 				'modules/comment-likes/comment-like-count.js'
 			),
 			array( 'jquery' ),
-			JETPACK__VERSION
+			JETPACK__VERSION,
+			$in_footer = false
 		);
 	}
 
+	/**
+	 * Adds like count column to admin page.
+	 *
+	 * @param array $columns - column of admin table.
```

Comment:
The dash isn't needed in phpdocs but I don't think it hurts to leave them.

Reply:
Yeah, I can never remember if it's JS or PHP that we need the dash so my brain just does it for both now.

Reply:
Personally I'd avoid the dash in PHP code, to match all the rest of the PHP code.